### PR TITLE
Fix Localization

### PR DESCRIPTION
**Pull request by PurpleHato**
FIX: French Localization (Looks like it's okay after 5 small tests)
TWEAK: shard of agony -> stone of agony
TWEAK: odd poultice -> odd potion

### CODE REVIEW

File: soh/soh/Enhancements/randomizer/randomizer.cpp [hunk 1]
```diff
@@ -1352,7 +1352,7 @@ std::unordered_map<std::string, RandomizerGet> SpoilerfileGetNameToEnum = {
     { "Acheter: Missiles", RG_BUY_BOMBCHU_5 },
     { "Buy Deku Seeds (30)", RG_BUY_DEKU_SEEDS_30 },
     { "Acheter: Graines Mojo (30)", RG_BUY_DEKU_SEEDS_30 },
-    { "Sold Out", RG_SOLD_OUT },
```
Comment (briaguya0) [inline soh/soh/Enhancements/randomizer/randomizer.cpp:1355]: just making sure we still have this in english too

**Review by briaguya0 — CHANGES_REQUESTED** [review]
overall it's looking great! super exciting stuff. as soon as the shard/poultice enum names are changed i think it's good to merge (assuming all the names are matching)

File: soh/soh/Enhancements/randomizer/randomizer.cpp (hunk 1 above)
Comment (PurpleHato) [reply soh/soh/Enhancements/randomizer/randomizer.cpp:1355]: Whoopsie! Done!

**Review by briaguya0 — COMMENTED** [review]
addresses https://github.com/briaguya-ai/rando-issue-tracker/issues/9 and https://github.com/briaguya-ai/rando-issue-tracker/issues/16

**Pull request merged**